etc/sample.opennebula.yaml

```diff
@@ -87,8 +87,9 @@ compute:
     shares:
         # the vo name
         ops:
-            # Group name in OpenNebula, default is equal to share name
-            #project: ops
+            #auth:
```

Comment:
Here we can probably use a more complete example similar to what is done for the OpenStack sample.
Please also update https://github.com/EGI-Foundation/cloud-info-provider/blob/master/etc/sample.opennebularocci.yaml#L85 accordingly.

Reply:
>Please also update https://github.com/EGI-Foundation/cloud-info-provider/blob/master/etc/sample.opennebularocci.yaml#L85 accordingly.

done

Reply:
Thanks @orviz, could you please uncomment the `auth` part for the ops VO? I don't see a reason why it shouldn't be similar to other samples, sorry I wasn't clear enough.